controller_manager/doc/userdoc.rst

```diff
@@ -228,3 +228,9 @@ Restarting hardware
 If hardware gets restarted then you should go through its lifecycle again.
 This can be simply achieved by returning ``ERROR`` from ``write`` and ``read`` methods of interface implementation.
 **NOT IMPLEMENTED YET - PLEASE STOP/RESTART ALL CONTROLLERS MANUALLY FOR NOW** The controller manager detects that and stops all the controllers that are commanding that hardware and restarts broadcasters that are listening to its states.
```

Comment:
Isn't this section outdated?

Reply:
I belive yes, it is. If you prefer, I can remove it

Reply:
Done